### ISSUE TITLE
Add spacing above document title

### ISSUE THREAD

**Issue by ajgrf** [minad/org-modern#143]
I'm using [visual-fill-column](https://codeberg.org/joostkremers/visual-fill-column) to add left and right margins, and to center my Org documents in the window. This makes it look odd that the title is pushed up against the top of the screen. [^1]

It would be nice if org-modern had an option to add spacing above the document title to sort of mimic a top margin. This can be done by setting the `line-height` text property on the newline following the title.

[^1]: Prot's new [spacious-padding](https://protesilaos.com/codelog/2023-06-03-emacs-spacious-padding/) package achieves a top margin by increasing the internal border width, but its changes are global and too invasive for me, affecting the mode line, echo area, tab bar, fringe, and all buffers.

**Comment by minad** [comment]
I agree that this looks odd, but you get the same effect if you scroll around. Then the content will also lack breathing room to the top. Therefore I am not convinced that we should add such additional spacing to org-modern for the title only. Personally I use a similar style as in Rougier's nano-emacs or Prot's package, where I increase the internal frame border and window divider widths. It looks nice but it is a bit wasteful or intrusive as you say.

**Comment by ajgrf** [comment]
Fair enough, it's easy enough to do this myself. I just thought it might fit into org-modern, since it's just another text property for Org files.

Thanks for all the work you do on your Emacs packages, by the way! I use your whole suite of vertico packages, corfu, and org-modern every day, and I'm considering adding jinx to my config as well.

**Comment by minad** [comment]
Thanks! To be clear, I welcome such feature suggestions. But I also have to decide which features are beneficial to most users and which rather cover special edge cases. Given Emacs customizability one can argue in favor of adding more and more features guarded by customization variables, but one should not forget that this also makes the system increasingly harder to get into. If special edge case features can be easily implemented on the level of the user configuration (hooks, advices, additional font lock rules, ...) it may be better to not add them to a package.
